package.json

```diff
@@ -40,10 +40,10 @@
     "vue-property-decorator": "^9.0.2"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.4.0",
-    "@typescript-eslint/parser": "^4.4.0",
-    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0",
-    "eslint-plugin-vue": "^5.2.3 || ^6.0.0 || ^7.0.0"
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^8.0.0",
```

Comment:
Should align with `@typescript-eslint/*` (`"eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"`) and `eslint-plugin-vue` (`"eslint": "^6.2.0 || ^7.0.0 || ^8.0.0"`) here

Reply:
Right: updated to: `"eslint": "^6.2.0 || ^7.0.0 || ^8.0.0"`